open-sphere-base/core/src/main/java/io/opensphere/core/control/keybinding/ControlTab.java

```diff
@@ -0,0 +1,194 @@
+        col2Constraints.setPercentWidth(6);
+        ColumnConstraints col3Constraints = new ColumnConstraints();
+        col3Constraints.setPercentWidth(47);
+        myDisplayArea.getColumnConstraints().addAll(col1Constraints, col2Constraints, col3Constraints);
+        myDisplayArea.setVgap(5);
+        myDisplayArea.setPadding(new Insets(0, 10, 10, 10));
+        HBox topheader = createHeader(header);
+        GridPane.setHgrow(topheader, Priority.ALWAYS);
+        thePane.add(topheader, 0, 0);
+        thePane.add(myDisplayArea, 0, 1);
+
+        return thePane;
+    }
+
+    public GridPane getDisplayArea()
+    {
+        return myDisplayArea;
+    }
+
+    public void setDisplayArea(GridPane myDisplayArea)
+    {
+        this.myDisplayArea = myDisplayArea;
+    }
+
+    /**
+     * @param theLabel the text to set.
+     * @return mainHeader the HBox containing the sub window title.
+     */
+    public HBox createHeader(String theLabel)
+    {
+        HBox mainHeader = new HBox();
+        mainHeader.setStyle("-fx-background-color : #181a1c");
+        Label header = new Label(theLabel);
+        header.setFont(Font.font(16));
+        mainHeader.getChildren().add(header);
+        mainHeader.setAlignment(Pos.CENTER_LEFT);
+        HBox.setHgrow(header, Priority.ALWAYS);
+        return mainHeader;
+    }
+
+    /**
+     * @param b1 the text to be set.
+     * @return comboText styled HBox containing text or a styled button.
+     */
+    public HBox createCustomText(String b1)
+    {
+        HBox comboText = new HBox();
+        comboText.setMinHeight(30);
+        comboText.setSpacing(8);
+        comboText.setAlignment(Pos.CENTER_LEFT);
+        if (!b1.equals(""))
+        {
+            comboText.getChildren().add(new IconDispButton(b1, false));
+        }
+        return comboText;
+    }
+
+    /**
+     * @param string the image to load.
+     * @return customView an Image display with a locked size and ratio.
+     */
+    public ImageView customImageView(String string)
+    {
+        ImageView customView = new ImageView(string);
+        customView.setFitHeight(30);
+        customView.setFitWidth(30);
+        customView.setPreserveRatio(true);
+        return customView;
+    }
+
+    /**
+     * @param label the text to be set.
+     * @return theLabelBox a VBox containing the text, right justified.
+     */
+    public VBox createLeftLabel(String label)
+    {
+        VBox theLabelBox = new VBox();
+        Label text2 = new Label(label);
+        theLabelBox.getChildren().add(text2);
+        theLabelBox.setAlignment(Pos.CENTER_RIGHT);
+        return theLabelBox;
+    }
+
+    /**
+     * A custom button for putting text or images into buttons.
+     */
+    public class IconDispButton extends Button
+    {
+        public IconDispButton(String key, boolean image)
+        {
+            setDisable(true);
+            setStyle("-fx-background-color:#0066cc;-fx-opacity: 1;");
+            if (image == false)
```

Comment:
Since image a boolean, you can just use "!image".
For readability, I would swap the two bodies, and have the if statement start with the true statement

Reply:
ended up removing since never used anyway